src/components/elements/Inputs/InputElements/InputWrapper.tsx

```diff
@@ -11,21 +13,41 @@ export interface InputWrapperProps {
   inputId?: string;
   label?: string;
   description?: string;
+  descriptionFooter?: string;
   containerClassName?: string;
+  labelVariant?: TextVariants;
   error?: FieldError;
   required?: boolean;
   feedbackRequired?: boolean;
+  labelClassName?: string;
+  descriptionClassName?: string;
+  sufixLabelView?: boolean;
+  classNameError?: string;
 }
 
 const InputWrapper = (props: PropsWithChildren<InputWrapperProps>) => {
   return (
     <div className={classNames(props.containerClassName, "relative space-y-2")}>
-      <InputLabel htmlFor={props.inputId} required={props.required} feedbackRequired={props.feedbackRequired}>
+      <InputLabel
+        htmlFor={props.inputId}
+        required={props.required}
+        feedbackRequired={props.feedbackRequired}
+        labelVariant={props.labelVariant}
+        className={props.labelClassName}
+        sufixLabelView={props.sufixLabelView}
+      >
         {props.label}
       </InputLabel>
-      <InputDescription id={props.inputId && `${props.inputId}-description`}>{props.description}</InputDescription>
+      <InputDescription id={props.inputId && `${props.inputId}-description`} className={props.descriptionClassName}>
+        {props.description}
```

Comment:
I'm surprised props.label above wasn't using `t`. Are the consumers of this component already translating? Please investigate and use `t` for label and description in here, or make sure the consumers are doing it before passing the values in.

Reply:
The label and description are being translated as they are in the SignUpForm component.
      <Input
        name="job_role"
        formHook={form}
        error={errors.job_role}
        type="text"
        label={t("Job Title")}
        description={t("Please enter your job role or position within your organization.")}
        required
      />